packages/context/src/interceptor.ts

```diff
@@ -200,15 +206,28 @@ export class InvocationContext extends Context {
 /**
  * The `BindingTemplate` function to configure a binding as a global interceptor
  * by tagging it with `ContextTags.INTERCEPTOR`
- * @param binding - Binding object
+ * @param group - Group for ordering the interceptor
  */
 export function asGlobalInterceptor(group?: string): BindingTemplate {
   return binding => {
-    binding.tag(ContextTags.GLOBAL_INTERCEPTOR);
+    binding
+      // Tagging with `GLOBAL_INTERCEPTOR` is required.
+      .tag(ContextTags.GLOBAL_INTERCEPTOR)
+      // `GLOBAL_INTERCEPTOR_NAMESPACE` is to make the binding key more readable.
+      .tag({[ContextTags.NAMESPACE]: GLOBAL_INTERCEPTOR_NAMESPACE});
     if (group) binding.tag({[ContextTags.GLOBAL_INTERCEPTOR_GROUP]: group});
   };
 }
 
+/**
+ * `@globalInterceptor` decorator to mark a class a global interceptor
```

Comment:
Missing "as"? English is not my mother tongue, I may be wrong.

```suggestion
 * `@globalInterceptor` decorator to mark a class as a global interceptor
```